client/app/bundles/HelloWorld/actions/helloWorldActionCreators.jsx

```diff
@@ -0,0 +1,6 @@
+import actionTypes from '../constants/helloWorldConstants';
+
+export const updateName = (text) => ({
+  type: actionTypes.HELLO_WORLD_NAME_UPDATE,
+  text,
+});
```

Comment:
Linter will warn that I should use `export default` here as there's only one action. But if I did so, mapDisptachToProps related code in the container file (HelloWorld.jsx) would be weird. Can I create a dummy action here to bypass this warning?